### ISSUE TITLE
Many error messages in tests on Python 3.4.3

### ISSUE THREAD

**Issue by tvoinarovskyi** [aio-libs/aiohttp#321]
In Python 3.4.3 when running tests you get allot of errors like this one:

```
Exception ignored in: Exception ignored in: <generator object start at 0x105561900>
Traceback (most recent call last):
  File "/Users/taras/workspace/aiohttp/aiohttp/server.py", line 275, in start
    self.transport.close()
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/asyncio/selector_events.py", line 568, in close
    self._loop.call_soon(self._call_connection_lost, None)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/asyncio/base_events.py", line 427, in call_soon
    handle = self._call_soon(callback, args)
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/asyncio/base_events.py", line 436, in _call_soon
    self._check_closed()
  File "/Library/Frameworks/Python.framework/Versions/3.4/lib/python3.4/asyncio/base_events.py", line 265, in _check_closed
    raise RuntimeError('Event loop is closed')
RuntimeError: Event loop is closed
```

Seems like some finalisations are missing in tests.

**Comment by fafhrd91** [comment]
thats known issue.

**Comment by asvetlov** [comment]
Fixed in #324
